bibliography.bib

```diff
@@ -143,6 +143,18 @@ @article{astropy2018
 	abstract = {The Astropy Project supports and fosters the development of open-source and openly developed Python packages that provide commonly needed functionality to the astronomical community. A key element of the Astropy Project is the core package astropy, which serves as the foundation for more specialized projects and packages. In this article, we provide an overview of the organization of the Astropy project and summarize key features in the core package, as of the recent major release, version 2.0. We then describe the project infrastructure designed to facilitate and support development for a broader ecosystem of interoperable packages. We conclude with a future outlook of planned new features and directions for the broader Astropy Project.}
 }
 
+@BOOK{NAP2019,
+  author    = "National Academies of Sciences, Engineering, and Medicine",
```

Comment:
The build is failing for some bibtex-related reason it seems. I think you need to wrap the author name in curly braces, .e.g. `"{National Academies of Sciences, Engineering, and Medicine}"`

Reply:
Ah, you are right -- thanks!